src/SignalR/clients/java/signalr/src/main/java/com/microsoft/signalr/HttpHubConnectionBuilder.java

```diff
@@ -113,12 +115,25 @@ public HttpHubConnectionBuilder withHeader(String name, String value) {
         return this;
     }
 
+    /**
+     * Adds a method that will be called when constructing the HttpClient to allow customization such as certificate validation, proxies, and cookies.
```

Comment:
We should probably call out that calling configureHttpClient multiple times overrides the previous configureBuilder.

Reply:
Maybe it should run all

Reply:
If the last set wins then the method name should be `setConfigureHttpClient(...)`. The comment would be "~Adds~ Sets a method that"